AutoWorkup/utilities/misc.py

```diff
@@ -26,6 +26,22 @@ def CommonANTsRegistrationSettings(antsRegistrationNode,
     """
     ## TODO: Consider registration masking
 
+    if ( registrationTypeDescription == "FiveStageAntsRegistrationT1Only" ) or ( registrationTypeDescription == "FiveStageAntsRegistrationMultiModal" ):
+        local_num_stages=5 # Assumes BLI landmark initialization
```

Comment:
Assumes BLI Affine landmark initialization, so don't do an initial Rigid registration to start.